components/eco-bank/EcoBankDetail.vue

```diff
@@ -52,7 +52,7 @@
       <template #products>
         <div class="grid grid-cols-1 md:grid-cols-3 gap-4 md:-mr-24">
           <p>
-            <strong>Current Accounts:</strong>
+            <strong>{{ isBE() ? 'Checking' : 'Current' }} Accounts:</strong>
```

Comment:
I think this should be the other way around, right? 
```suggestion
            <strong>{{ isBE() ? 'Current' : 'Checking' }} Accounts:</strong>
```